network/genesis.go

```diff
@@ -33,6 +34,7 @@ func LoadLocalGenesis() (map[string]interface{}, error) {
 	corethCChainGenesis := coreth_params.AvalancheLocalChainConfig
 	if _, ok := os.LookupEnv("CAMINO_NETWORK"); ok {
 		corethCChainGenesis.SunrisePhase0BlockTimestamp = big.NewInt(0)
+		corethCChainGenesis.ChainID = big.NewInt(431120)
```

Comment:
I suggest moving this value in a separate file containing constants. eg under /util/constants/camino_constants.go and move the value there. Add a comment why it's necessary. And maybe give a value relative to the 
`AvalancheLocalChainID`  in caminoethvm/params/config.go  (eg LocalID - 1) to hint that it replaces this local chain id.
or alternatively the value '503' matching this logic

![image](https://user-images.githubusercontent.com/14131271/229715976-41cae592-3a3b-4d55-9be1-c60b5e3f20fa.png)